package.json

```
@@ -48,6 +48,8 @@
     "@nestjs/common": "^9.0.0",
     "@nestjs/core": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
+    "@nestjs/swagger": "^6.1.3",
+    "@nicholas.braun/nestjs-redoc": "^2.2.3",
```

Let's lock this at `2.2.3`

```suggestion
    "@nicholas.braun/nestjs-redoc": "2.2.3",
```